src/main/scala/com/cognite/sdk/scala/v1/dataModelProperties.scala

```diff
@@ -14,3 +14,39 @@ final case class Float64Property(value: Double) extends PropertyTypePrimitive
 final case class StringProperty(value: String) extends PropertyTypePrimitive
 
 final case class ArrayProperty[+A <: PropertyTypePrimitive](values: Vector[A]) extends PropertyType
+
+// These types below are treated as string for now
+final case class DirectRelationProperty(value: String) extends PropertyType
+final case class TimeStampProperty(value: String) extends PropertyType
+final case class DateProperty(value: String) extends PropertyType
```

Comment:
Also this and interval if we introduce it

Reply:
it's not available in the api yet so I think we can handle the interval type later